src/devices/BrickPi3/samples/SensorTests.cs

```diff
@@ -1,18 +1,18 @@
-﻿using Iot.Device.BrickPi3.Models;
-using Iot.Device.BrickPi3.Sensors;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
there should be also extra enter after the copyrightheader (not sure if stylecop has any rule for that)

Reply:
I wouldn't know. Normally it's even forbidden to have empty lines after comments, but the header seems to be an exception.